tests/tasks/test_tasks.py

```diff
@@ -17,18 +19,21 @@
 
 
 def create_task(tasks_body):
+    """Creates task with the given task body."""
```

Comment:
Just a comment: For actual package code (not tests), we would want to go for full Google-style docstrings with sections for `Args`, `Attributes`, `Returns` and `Raises` etc. We'd also want to give type hints at least for args and return values. Just like in proTES.

Reply:
I'll keep that in mind thanks!